common/src/main/java/tachyon/Version.java

```diff
@@ -23,15 +28,43 @@
 public final class Version {
```

Comment:
What was this "Version" class previously used for? Is it confusing that the same "Version" class is used for the "0.8.0" style of versioning, and the master service style of versioning?